library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/processor/WireframeUtils.kt

```diff
@@ -86,7 +86,7 @@ internal class WireframeUtils {
     private fun MobileSegment.ShapeStyle.isOpaque(): Boolean {
         return this.opacity == FULL_OPACITY_ALPHA &&
             this.backgroundColor != null &&
-            this.backgroundColor.takeLast(2) == FULL_OPACITY_ALPHA_AS_HEXA_STRING
+            this.backgroundColor.takeLast(2).lowercase() == FULL_OPACITY_STRING_HEXA
```

Comment:
```suggestion
            this.backgroundColor.takeLast(2).lowercase(Locale.US) == FULL_OPACITY_STRING_HEXA
```

Reply:
Isn't it by default ?

Reply:
Oh the default is the `ROOT` meaning the project locale....damn